android/build.gradle

```diff
@@ -1,24 +1,37 @@
-buildscript {
-    ext.kotlin_version = '1.6.20'
+allprojects {
     repositories {
         google()
         mavenCentral()
     }
-
-    dependencies {
-        classpath 'com.android.tools.build:gradle:7.1.2'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-    }
 }
 
-allprojects {
-    repositories {
-        google()
-        mavenCentral()
+rootProject.buildDir = '../build'
+
+
+// [NStack] the namespace property is null within 
+// the android block of the build.gradle file for the nstack sdk.
+// Additionally, the Kotlin Gradle plugin is using a very old version there.
+subprojects {
+    afterEvaluate { project ->
+        if (project.name == 'nstack') {
+            if (project.hasProperty('android')) {
+                project.android {
+                    if (namespace == null) {
+                        namespace project.group
+                    }
+                }
+                project.buildscript {
+                    dependencies {
+                        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.10"
+                    }
+                }
+            }
+        }
```

Comment:
The [Nstack PR](https://github.com/nstack-io/flutter-sdk/pull/57) is ready for review, maybe we can wait for it to be merged and remove this workaround

Reply:
Ok, i will keep track of this, thanks @hassan-saleh-ml